synthtool/gcp/templates/node_library/.kokoro/test.sh

```diff
@@ -21,6 +21,17 @@ export NPM_CONFIG_PREFIX=/home/node/.npm-global
 cd $(dirname $0)/..
 
 npm install
+# If tests are running against master, configure Build Cop
+# to open issues on failures:
+if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]]; then
+  export MOCHA_REPORTER_OUTPUT=test_output_sponge_log.xml
```

Comment:
build cop \o/, note that we set the reporter only in `continuous` (landing to `master`), which outputs our tests in a format that can be piped.